### DISCUSSION TITLE
Blog-Doc: The Simplest Node.js CMS & SSG!

### DISCUSSION THREAD

**Discussion by LebCit** [markedjs/marked#2911] [Show and tell]
Hello,

I've just rebuilt [Blog-Doc](https://github.com/LebCit/blog-doc) from ground-up using Eta, Hono and Marked.
You can read more about the new version of Blog-Doc in the following article: [From 145 to 7 💪](https://lebcit.github.io/posts/from-145-to-7/)

A big thanks to all Marked developers for their time and efforts, bringing a 0 dependency Markdown parser and compiler.

**Comment by UziTech** [comment]
If you want you can add that to the [List of Tools Using Marked](https://github.com/markedjs/marked/blob/master/docs/INDEX.md?plain=1#L122)

**Reply by LebCit** [reply]
Thank you @UziTech 
Of course I'm interested in listing Blog-Doc as an app using marked.
I really prefer that anyone from marked put it wherever it seems to be better in the list 😃 
Thanks again.
